doc/manuals/admin/build_source.md

```diff
@@ -51,9 +51,9 @@ commands that require root privilege):
         sudo make install  # installation puts .h files in /usr/local/include and library in /usr/local/lib
         sudo ldconfig      # just in case... it doesn't hurt :)
 
-* Install Google Test/Mock from sources (there are RPM pacakges for this, but they don't seem to be working with the current CMakeLists.txt configuration). Previously the URL was http://googlemock.googlecode.com/files/gmock-1.5.0.tar.bz2 but google removed that package in late August 2016 and it is no longer working.
+* Install Google Test/Mock from sources (there are RPM packages for this, but they don't seem to be working with the current CMakeLists.txt configuration). Previously the URL was http://googlemock.googlecode.com/files/gmock-1.5.0.tar.bz2 but google removed that package in late August 2016 and it is no longer working.
```

Comment:
s/google/Google

s/don't seem/do not work/

Reply:
google removed -> Google removed

Reply:
fixed in c6e45d153045d7d51e08a4dc3c732985c22fe5c9